opencog/benchmark/AtomSpaceBenchmark.cc

```diff
@@ -327,9 +327,9 @@ void AtomSpaceBenchmark::startBenchmark(int numThreads)
             if (pymo == NULL) pymo = new PythonModule(*cogs);
             pymo->init();
             asp = &cogs->getAtomSpace();
-            pyev = &PythonEval::instance(asp);
```

Comment:
python eval should work with the atompsace that it is given. It cannot assume a default atomspace, there may be several.  Yes I understand the current design might not be able to do this, but I thought I should mention it.